diff -Nur -x '*.orig' -x '*.rej' e-smith-pptpd-1.12.0/root/etc/e-smith/templates/etc/pptpd.conf/debug mezzanine_patched_e-smith-pptpd-1.12.0/root/etc/e-smith/templates/etc/pptpd.conf/debug
--- e-smith-pptpd-1.12.0/root/etc/e-smith/templates/etc/pptpd.conf/debug	2002-05-18 14:15:06.000000000 -0400
+++ mezzanine_patched_e-smith-pptpd-1.12.0/root/etc/e-smith/templates/etc/pptpd.conf/debug	2006-04-09 15:55:02.000000000 -0400
@@ -1,7 +1,4 @@
 {
-# TAG: debug
-#
-#	Turns on (more) debugging to syslog.
-#
+    my $debug = $pptpd{debug} || 'no';
+    $OUT = $debug eq 'yes' ? "debug" : "# debug is not enabled";
 }
-debug
