diff -Nur -x '*.orig' -x '*.rej' e-smith-pptpd-1.11.0/root/etc/e-smith/db/configuration/defaults/pptpd/access mezzanine_patched_e-smith-pptpd-1.11.0/root/etc/e-smith/db/configuration/defaults/pptpd/access
--- e-smith-pptpd-1.11.0/root/etc/e-smith/db/configuration/defaults/pptpd/access	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_e-smith-pptpd-1.11.0/root/etc/e-smith/db/configuration/defaults/pptpd/access	2005-07-20 19:11:52.925872144 -0600
@@ -0,0 +1 @@
+public
diff -Nur -x '*.orig' -x '*.rej' e-smith-pptpd-1.11.0/root/etc/e-smith/db/configuration/defaults/pptpd/sessions mezzanine_patched_e-smith-pptpd-1.11.0/root/etc/e-smith/db/configuration/defaults/pptpd/sessions
--- e-smith-pptpd-1.11.0/root/etc/e-smith/db/configuration/defaults/pptpd/sessions	2003-03-07 12:35:41.000000000 -0700
+++ mezzanine_patched_e-smith-pptpd-1.11.0/root/etc/e-smith/db/configuration/defaults/pptpd/sessions	2005-07-20 19:12:19.931466496 -0600
@@ -1 +1 @@
-10
+0
diff -Nur -x '*.orig' -x '*.rej' e-smith-pptpd-1.11.0/root/etc/e-smith/db/configuration/defaults/pptpd/TCPPort mezzanine_patched_e-smith-pptpd-1.11.0/root/etc/e-smith/db/configuration/defaults/pptpd/TCPPort
--- e-smith-pptpd-1.11.0/root/etc/e-smith/db/configuration/defaults/pptpd/TCPPort	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_e-smith-pptpd-1.11.0/root/etc/e-smith/db/configuration/defaults/pptpd/TCPPort	2005-07-20 19:11:01.330535760 -0600
@@ -0,0 +1 @@
+1723
diff -Nur -x '*.orig' -x '*.rej' e-smith-pptpd-1.11.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/45AllowPPTP mezzanine_patched_e-smith-pptpd-1.11.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/45AllowPPTP
--- e-smith-pptpd-1.11.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/45AllowPPTP	2002-08-28 15:21:57.000000000 -0600
+++ mezzanine_patched_e-smith-pptpd-1.11.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/45AllowPPTP	1969-12-31 17:00:00.000000000 -0700
@@ -1,5 +0,0 @@
-{
-    my $status = $pptpd{status} || 'disabled';
-
-    $OUT = allow_tcp_in(1723, ($status eq 'enabled' ));
-}
