diff -Nur -x '*.orig' -x '*.rej' e-smith-pptpd-1.12.0/root/etc/e-smith/templates/etc/ppp/options.pptpd/mtu mezzanine_patched_e-smith-pptpd-1.12.0/root/etc/e-smith/templates/etc/ppp/options.pptpd/mtu
--- e-smith-pptpd-1.12.0/root/etc/e-smith/templates/etc/ppp/options.pptpd/mtu	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_e-smith-pptpd-1.12.0/root/etc/e-smith/templates/etc/ppp/options.pptpd/mtu	2008-01-12 13:36:40.000000000 -0700
@@ -0,0 +1 @@
+mtu 1400
diff -Nur -x '*.orig' -x '*.rej' e-smith-pptpd-1.12.0/root/etc/e-smith/templates/etc/ppp/options.pptpd/mru mezzanine_patched_e-smith-pptpd-1.12.0/root/etc/e-smith/templates/etc/ppp/options.pptpd/mru
--- e-smith-pptpd-1.12.0/root/etc/e-smith/templates/etc/ppp/options.pptpd/mru	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_e-smith-pptpd-1.12.0/root/etc/e-smith/templates/etc/ppp/options.pptpd/mru	2008-01-12 13:36:40.000000000 -0700
@@ -0,0 +1 @@
+mru 1400
